.gitignore

```diff
@@ -2,3 +2,4 @@
 doc
 coverage
 rmt.pot
+.idea/*
```

Comment:
It would be better to keep it in the local `.gitignore` file, as not everybody uses it.